domainlab/exp_protocol/run_experiment.py

```diff
@@ -77,6 +77,7 @@ def run_experiment(
     apply_dict_to_args(args, args_algo_as_task)
     apply_dict_to_args(args, hyperparameters)
     apply_dict_to_args(args, misc, extend=True)
+    args.device = str(param_index % 4)
```

Comment:
I'm a little confused now. What does this change?

Does this mean it is now hardcoded to use 4 GPU?

Reply:
Oh, I just read the title of the pull request

Reply:
Denbi only has 1 GPU right?